tests/ui/stats/hir-stats.rs

```diff
@@ -1,7 +1,7 @@
 // check-pass
 // compile-flags: -Zhir-stats
 // only-x86_64
-
+// ignore-stage1
```

Comment:
If this is needed temporarily, can you add a comment saying that?

Reply:
write this as a cfg(not(bootstrap)) so that it gets picked up during the version bump